components/brave_wallet/browser/brave_wallet_prefs.cc

```diff
@@ -277,6 +274,11 @@ void MigrateObsoleteProfilePrefs(PrefService* prefs) {
 
   // Added 02/2023
   TxStateManager::MigrateAddChainIdToTransactionInfo(prefs);
+
+  // Added 04/2023
+  registry->RegisterTimePref(kBraveWalletP3ALastReportTime, base::Time());
+  registry->RegisterTimePref(kBraveWalletP3AFirstReportTime, base::Time());
+  registry->RegisterListPref(kBraveWalletP3AWeeklyStorage);
```

Comment:
We can't register local state pref in profile pref registration.
Please expose a new function like I mentioned in https://github.com/brave/brave-core/pull/17649#discussion_r1156702414